### DISCUSSION TITLE
mod addition i think would be well worth it

### DISCUSSION THREAD

**Discussion by Lil-Gangrene** [NillerMedDild/Enigmatica6#3554] [Ideas]
ik the developers for this mod pack get this a lot but I reckon the End Remastered mod would be a great addition for this mod pack. it redid the way you get to the end by making it so there are different eye types that you have to collect by doing various things like villager trade, wither fight, and dungeon loot in different biomes and combine them all to activate the portal. it also replaces strongholds with end castles, and a remade version of the stronghold each with mobs and puzzles that are on vastly bigger scales than vanilla. it also makes tweaks to the ender dragon fight itself to make it more of a challenge.  I think this would be good because it makes getting to the end fun and not just a chore to do within the first IRL day or 2 of playing and it would work well with the quest system and other implementations currently in Enigmatic 6 and it fully fleshed out not just in its early stages of development.

https://www.curseforge.com/minecraft/mc-mods/endremastered/files

**Comment by NielsPilgaard** [comment]
Duplicate of https://github.com/NillerMedDild/Enigmatica6/issues/482 :)
